naptime-graphql/src/main/scala/org/coursera/naptime/ari/graphql/schema/NaptimePaginatedResourceField.scala

```diff
@@ -173,9 +173,8 @@ object NaptimePaginatedResourceField {
             idsWithStart.take(limit)
           }.getOrElse(List.empty)
         }
-        objects.collect {
-          case (id, element) if ids.contains(id) => element
-        }.toSeq
+        val objectMap = objects.map { case (id, element) => id -> element }.toMap
```

Comment:
are you able to do `objects.toMap` directly? i thought in scala that the -> was the same as just a 2-tuple